README.md

```diff
@@ -109,6 +111,8 @@ If you want to remove also the volumes associated with the container, the deleti
 
 There's also a [logging driver](https://docs.docker.com/engine/admin/logging/overview/) available for individual containers in docker 1.10. To run docker with a custom log driver (i.e., to syslog), use `docker run --log-driver=syslog`.
 
+Another useful option is `docker run --name yourname docker_image` because when you specify the --name inside the run command this will allow you to start and stop a container by calling it with the name the you specified when you created it.
```

Comment:
@H4M1O For consistency with other examples, I would add the formatting around `--name` so you'd have

> because when you specify the `--name` inside

instead of

> because when you specify the --name inside